src/telemetry/surveys/awareness.ts

```diff
@@ -7,14 +7,17 @@ import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { Survey } from './SurveyManager';
 
-const minimumOverallSessions = 3;
+const minimumOverallSessions = 2;
 const maximumNotEditOnlySessions = 0;
 
 export const awareness: Survey = {
     id: 'aware',
-    url: 'https://aka.ms/AA8lh3s',
     prompt: localize('vscode-docker.survey.aware.prompt', 'The Docker extension is always adding new features. Do you want to learn more about them?'),
-    buttons: [localize('vscode-docker.survey.aware.button.learnMore', 'Learn more'), localize('vscode-docker.survey.aware.button.never', 'Don\'t show again')],
+    buttons: new Map<string, string | undefined>([
+        [localize('vscode-docker.survey.aware.button.watch', 'Watch a video'), 'https://aka.ms/AA8lh3s'],

```

Comment:
TODO: strings, URLs

Reply:
@BigMorty for this

Reply:
https://aka.ms/AA9j59w is the link for "Read documentation" button.

Reply:
Agreed on "Watch a video" and "Read docs", looks like this:
![image](https://user-images.githubusercontent.com/36966225/92157970-34669880-edf9-11ea-9105-f887b0e799fa.png)